apps/web/package.json

```diff
@@ -83,13 +84,16 @@
     "sonner": "^1.7.0",
     "tailwind-merge": "^2.2.1",
     "utils": "link:@/lib/utils",
+    "uuid": "^11.0.4",
     "zod": "^3.23.8"
   },
   "devDependencies": {
     "@shikijs/rehype": "^1.1.6",
     "@shikijs/transformers": "^1.1.6",
     "@svgr/cli": "^8.1.0",
     "@tailwindcss/typography": "^0.5.10",
+    "@testing-library/dom": "^10.4.0",
+    "@testing-library/react": "^16.1.0",
```

Comment:
Here and below, added [recommended Vitest dev deps](https://nextjs.org/docs/app/building-your-application/testing/vitest#manual-setup).